code/modules/mob/living/carbon/human/life.dm

```diff
@@ -1188,7 +1188,8 @@
 	else
 		AdjustLoseBreath(2, bound_lower = 0, bound_upper = 3)
 		adjustOxyLoss(5)
-		adjustBruteLoss(1)
+		Paralyse(4)
+		adjustBruteLoss(2)

```

Comment:
Heart attacks used to deal 20 oxy, 3 losebreath, deal brain damage, and weaken; then they were nerfed to their current state (by me), now it's getting buffed up a bit, again.